packages/holocron/src/reactStreaming.js

```diff
@@ -0,0 +1,39 @@
+import { createContext, useContext } from 'react';
+
+const ReactStreamingContext = createContext({});
+
+const ModuleContext = createContext();
+
+const useAsyncModuleData = (key) => {
+  const moduleName = useContext(ModuleContext);
```

Comment:
Can the module context be an object with a name property

If we then imagine a new thing to place in the module context in the future, it's not a type change, it's just a new property

Reply:
Added in [`825e153` (#177)](https://github.com/americanexpress/holocron/pull/177/commits/825e1532d3ab4ac90f4b80f26eab137d39a1a490)